python/packages/azurefunctions/agent_framework_azurefunctions/_durable_agent_state.py

```diff
@@ -0,0 +1,506 @@
+# Copyright (c) Microsoft. All rights reserved.
+from __future__ import annotations
+
+import json
+
+from typing import Any, List, Dict, Optional, cast
+from datetime import datetime, timezone
+
+# Base content type
+
+class DurableAgentStateContent:
```

Comment:
Would you be able to add docstrings for each new class to explain why its needed please?